puppet/modules/jenkins_job_builder/files/centos.org/jobs/foreman-forklift-centos-devel-test.yml

```diff
@@ -0,0 +1,14 @@
+---
+- job:
+    name: 'foreman-forklift-centos-devel-test'
```

Comment:
Based on existing convention, I'd think this would be something more akin to `foreman-devel-test` or `foreman-katello-devel-test`

Reply:
I know it's not exactly in line with the convention, but as a developer, I would like to have a way to see which forklift boxes are broken with a quick search in jenkins. Ideally, we will copy this pattern for other forklift boxes (hammer-devel, foreman proxy, katello-client, etc...), so quickly searching them would be ideal. I am thinking adding 'forklift' in the title would be the best way to do this

Reply:
I get your idea, however, this does tightly couple them to a box idea, not even the pipeline (which is what is really being tested). This tends to tell me we need to revamp our pipeline / box naming rather than comprise on the job. Because the job reads confusingly to me in terms of what is actually being tested.

Reply:
how about `foreman-forklift-katello-devel-test`?

Reply:
I can live with that, but we should change all jobs to reflect this since they all come from some sort of forklift entity. We are striving for a consistent naming scheme across jobs for clarity.

Reply:
> we should change all jobs to reflect this since they all come from some sort of forklift entity.

@ehelms If I'm understanding this correctly, that brings me back to what I am looking for as a developer

> I would like to have a way to see which forklift boxes are broken with a quick search in jenkins.

So I would rather have something unique on each job that is tied to a box in some way, even if its indirectly through a meta role. I'm just thinking of how we interpret these jobs and find them useful as developers

Reply:
I guess the name is more important on theforeman.org servers, as that is where most developers would look (sorry, I didn't realize this is the centos file until now)

Would `forklift-katello-devel-test` work as the job name on the foreman jenkins server?

Reply:
I suppose so. This just doesn't follow our current conventions and ties the intent heavily to the implementation. I would think we care *that devel is being tested* not that it is being *tested within forklift*. If anything, to follow convention it would be more like `katello-devel-forklift-test` as that is specific to generic.